map/bookmark_manager.cpp

```diff
@@ -2818,6 +2822,12 @@ void BookmarkManager::MarksChangesTracker::OnUpdateMark(kml::MarkId markId)
     m_updatedMarks.insert(markId);
 }
 
+void BookmarkManager::MarksChangesTracker::OnUpdateLine(kml::TrackId lineId)
+{
+  if (m_createdLines.find(lineId) == m_createdLines.end())
```

Comment:
Checking created lines and inserting into edited lines below? Why?

Reply:
I used the same logic, as in the bookmarks to not break the overall behaviour.

```cpp
void BookmarkManager::MarksChangesTracker::OnUpdateMark(kml::MarkId markId)
{
  if (m_createdMarks.find(markId) == m_createdMarks.end())
    m_updatedMarks.insert(markId);
}

// new:
void BookmarkManager::MarksChangesTracker::OnUpdateLine(kml::TrackId lineId)
{
  if (m_createdLines.find(lineId) == m_createdLines.end())
    m_updatedLines.insert(lineId);
}
```

I don't feel capable of refactoring it myself. There is a lot of logic...